stl/src/parallel_algorithms.cpp

```diff
@@ -296,4 +296,4 @@ void __stdcall __std_execution_wake_by_address_all(const volatile void* _Address
 #endif // ^^^ _STL_WIN32_WINNT < _WIN32_WINNT_WIN8 ^^^
 }
 
-} // extern "C"
+EXTERN_C_END // extern "C"

```

Comment:
When changing this to the STL's macro, the comment should also be dropped (as it no longer serves a purpose).